### PR TITLE
Remove interpreter line execution to support Vagrant-Windows.

### PR DESCRIPTION
**Pull request by pmtischler**
Execute scripts with interpreter type.

For #28

### CODE REVIEW

**Comment by prattmic** [comment]
Don't actually remove the shebangs, they are useful when running manually, though removing them does provide a nice way to test.

Are there no Python scripts that need to be changed?

Note that the tests are failing.

**Comment by pmtischler** [comment]
I've put the shebangs back. I don't believe we call any python files or bash scripts directly, other than in Vagrant setup (or equivalently our doc for machine setup) and in the `test.sh` file.

It appears the tests are failing due to docs not building, which is happening for both PRs and seems to be a new error unrelated to this PR.

**Comment by prattmic** [comment]
`docs/setup/testing.rst` has direct invocations of test.sh.

**Comment by pmtischler** [comment]
Updated.

File: docs/setup/testing.rst [hunk 1]
```diff
@@ -10,7 +10,7 @@ To run the tests execute the following commands:
 
 .. code-block:: bash
 
-    $ ~/interop/test.sh
+    $ bash /interop/test.sh
 
 Or from outside Vagrant VM:
```
Comment (prattmic) [inline docs/setup/testing.rst]: The `vagrant ssh` invocation below also needs to be updated.

**Comment by prattmic** [comment]
Also `setup/README.md`

**Comment by pmtischler** [comment]
Done & Done.

**Comment by prattmic** [comment]
LGTM

**Pull request merged**